apps/explorer/src/pages/validator/ValidatorDataTypes.ts

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Can we just move this to sdk? I think I saw the same file in the wallet

Reply:
Jordan has a PR with this that has not landed. The plan is to remove this file one it land.